docs/networking/cilium.md

```diff
@@ -85,6 +85,23 @@ Note that since Cilium Operator is the entity that interacts with the EC2 API to
 
 Also note that this feature has only been tested on the default kops AMIs.
 
+#### Enabling Encryption in Cilium
+Starting kops verion 1.18, it is possible to enable encryption for Cilium agent.
```

Comment:
I think this feature will rather go into 1.19 due to the larger feature set we have already put into 1.18

Reply:
Fixed in https://github.com/kubernetes/kops/pull/9154/commits/41c76dabaad8ffcbef6a4b19dfdf1174325bd8f8